rllib/algorithms/sac/torch/sac_torch_learner.py

```diff
@@ -314,14 +310,23 @@ def compute_gradients(
             for component in (
                 ["qf", "policy", "alpha"] + ["qf_twin"] if config.twin_q else []
             ):
-                self.metrics.peek((module_id, component + "_loss")).backward(
+                # Get the optimizer for the current component and module.
```

Comment:
Nice! Did this help with learning performance a little?

Reply:
Yeah  it was somehow a combination with some hyperparameters.